src/collective/lazysizes/transform.py

```diff
@@ -9,14 +9,14 @@
 
 
 # grey rectangule, 16x16
-PLACEHOLDER = 'data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABAAAAAQAQMAAAAlPW0iAAAAA1BMVEXy8vJkA4prAAAAC0lEQVQI12MgEQAAADAAAWV61nwAAAAASUVORK5CYII='
+PLACEHOLDER = 'data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABAAAAAQAQMAAAAlPW0iAAAAA1BMVEXy8vJkA4prAAAAC0lEQVQI12MgEQAAADAAAWV61nwAAAAASUVORK5CYII='  # noqa
```

Comment:
I prefer to use the more specific `# noqa: E501` here

Reply:
you're right, but this goes against my developer laziness. I'll change it but I refuse to remember the E501 error code ;)

Reply:
the problem with using just `# noqa` is that it will hide other issues like, for instance, if we decide to later not to use this constant.